msu/hooks/scenarios/world/starting_scenario.nut

```diff
@@ -9,3 +9,22 @@
 	{
 	}
 });
+
+::mods_hookDescendants("scenarios/world/starting_scenario", function(o) {
+	if ("onUpdateLevel" in o)
+	{
+		local onUpdateLevel = o.onUpdateLevel;
+		o.onUpdateLevel = function( _bro )
+		{
+			onUpdateLevel(_bro);
+			_bro.getSkills().onUpdateLevel();
```

Comment:
Reverse the order to be consistent with how it'll behave with scenarios which don't have onUpdateLevel defined in them as MSU hook runs before everything else.